plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts

```diff
@@ -21,13 +21,22 @@ const test = base.extend< { checkoutPageObject: CheckoutPage } >( {
 test.describe( 'Shopper → Translations', () => {
 	test.beforeAll( async () => {
 		await cli(
-			`npm run wp-env run tests-cli -- wp language core activate nl_NL`
+			`npm run wp-env run tests-cli -- wp language core install nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp site switch-language nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin install woocommerce nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin update woocommerce`
```

Comment:
Is there a particular reason why we run this command?

Reply:
Good catch. I think I've initially added that part as after installing a different language within WP-Admin, I always go to the update page and update the translations. That said, as we're running `wp language plugin install woocommerce nl_NL` immediately before `wp language plugin update woocommerce`, I think the later command is indeed not necessary. I'll quickly try that.